Fix #1840: Introduce a nicer model for configuring network presentation. (#1848)
kroxylicious/kroxylicious: docs/modules/configuring/con-configuring-virtual-clusters.adoc

[role="_abstract"]
A Kafka cluster is represented by the proxy as a virtual cluster. 

> Clients talk to the virtual cluster rather than the actual cluster.

Reviewer: ```suggestion
Clients communicate with the virtual cluster rather than the actual cluster.
```
